src/ert/scheduler/job.py

```diff
@@ -67,6 +68,8 @@ def __init__(self, scheduler: Scheduler, real: Realization) -> None:
         self._scheduler: Scheduler = scheduler
         self._callback_status_msg: str = ""
         self._requested_max_submit: Optional[int] = None
+        self._start_time: Optional[float] = None
```

Comment:
why not datetime.datetime?

Reply:
maybe faster

Reply:
It all boils down to floats, but tbh tried datetime first but got a mypy error at some point.